flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/stream/sql/CalcITCase.scala

```diff
@@ -366,6 +366,24 @@ class CalcITCase extends StreamingTestBase {
     sink.getAppendResults.foreach(result => assertEquals(expected, result))
   }
 
+  @Test
+  def testIfFunction(): Unit = {
+
+    implicit val typeInfo: TypeInformation[Row] =
```

Comment:
The test will fail, 
```
org.apache.flink.table.api.ValidationException: SQL validation failed. At line 1, column 11: Column 'a' not found in any table
	at org.apache.flink.table.planner.calcite.FlinkPlannerImpl.org$apache$flink$table$planner$calcite$FlinkPlannerImpl$$validate(FlinkPlannerImpl.scala:187)
	at org.apache.flink.table.planner.calcite.FlinkPlannerImpl.validate(FlinkPlannerImpl.scala:113)
```
how about using 
```
    val testDataId = TestValuesTableFactory.registerData(TestData.data1)
    val ddl =
      s"""
         |CREATE TABLE t (
         |  a int,
         |  b varchar,
         |  c int
         |) WITH (
         |  'connector' = 'values',
         |  'data-id' = '$testDataId',
         |  'bounded' = 'true'
         |)
       """.stripMargin
    tEnv.executeSql(ddl)
    val expected = List("false,1", "false,2", "false,3", "true,4", "true,5", "true,6")
    val actual = tEnv
      .executeSql("SELECT IF(a > 3, 'true', 'false'), a from t")
      .collect()
      .map(r => r.toString)
      .toList
    assertEquals(expected.sorted, actual.sorted)
```?

Reply:
Ok, It would be more intuitive with DDL.